index.html

```diff
@@ -4,6 +4,8 @@
   <link type="text/css" rel="stylesheet" href="style.css" />
 </head>
 <body>
+  <div id="app">
```

Comment:
Try using a section tag instead of a div.  Semantic tags are much easier to trace errors with than an entire page filled with divs.  If there isn't a tag for what you are doing, make your own.  HTML tags are simply tags that are paired.